lib/puppet/application/lookup.rb

```diff
@@ -33,9 +33,12 @@ class Puppet::Application::Lookup < Puppet::Application
 
   option('--merge_hash_arrays')
 
-  # not yet supported
   option('--explain')
 
+  option('--explain_json')
```

Comment:
Other commands in puppet use a separate `--format=f` flag to set the format of the output. I think this command should do the same.